.github/workflows/publish-to-odr.yml

```diff
@@ -61,13 +61,13 @@ jobs:
         id: modified-files
         run: |
           # AM = Include: Added, Modified
-          echo "modified_files=$(git diff --name-only --diff-filter=AM ${{ github.event.before }} ${{ github.event.after }})" >> $GITHUB_OUTPUT
+
+          MODIFIED_FILEs=$(git diff --name-only --diff-filter=AM ${{ github.event.before }} ${{ github.event.after }})
+          MODIFIED_PARAMETERS_FILES=$(echo $MODIFIED_FILES | grep "publish-odr-parameters/" | grep ".yaml\$" | sort | uniq)
+          echo "modified_files=${MODIFIED_PARAMETERS_FILES}" >> $GITHUB_OUTPUT
```

Comment:
This value is only used in the subsequent step, so is it actually necessary to use `GITHUB_OUTPUT`?

Reply:
Yep. Basically, adding `GITHUB_OUTPUT` to the mix complicates things a fair bit, without adding any particular value other than splitting the collecting of paths from their use. It's a trade-off, but I think that means we should probably put them together.